globals.go

```diff
@@ -460,6 +460,7 @@ const (
 	HostCreateCommandName    = "host.create"
 	S3PushCommandName        = "s3.push"
 	S3PullCommandName        = "s3.pull"
+	ShellExecCommandName     = "shell.exec"
```

Comment:
Can you also use this where the command is created, for consistency? https://github.com/evergreen-ci/evergreen/blob/293d1e320dccba8f2e8675afa328e51d9d286102/agent/command/registry.go#L55 and https://github.com/evergreen-ci/evergreen/blob/9a855967091f8f17a1a63b66e3c2d8b22cd2265a/agent/command/shell.go#L80